pkg/manifest/manifest.go

```diff
@@ -1007,7 +1007,7 @@ func ReadManifest(cxt *portercontext.Context, path string) (*Manifest, error) {
 
 	m, err := UnmarshalManifest(cxt, data)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "unsupported property set or a custom action is defined incorrectly")
```

Comment:
Do we want to switch from using 'pkg/errors` here to `fmt.Errorf` ? so we can move closer to this goal https://github.com/getporter/porter/issues/2009